### PR TITLE
Fix - Assignes university role to User

### PR DESCRIPTION
**Pull request by julieahlquist**
PR to add university role to User in order to complete following user story:
````
As a University,
In order to use Gnosis as a service,
I would like to be able to sign up for an account and receive RG keys.
````
Link to chore Pivotal Tracker: https://www.pivotaltracker.com/n/projects/2384164/stories/167961196
Link to user story Pivotal Tracker: https://www.pivotaltracker.com/story/show/167867978

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -53,5 +53,21 @@
         it { is_expected.to allow_value(email).for(:email) }
       end
     end
+
+    describe 'User roles' do
+      let(:university){create :user, email: 'oxford@oxford.edu', role: :university}
```
Comment (oliverochman) [inline spec/models/user_spec.rb]: change variable name to `user` to make it more readable

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -53,5 +53,21 @@
...
+
```
Comment (oliverochman) [inline spec/models/user_spec.rb]: You need to test the `research_group` role as well.

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: spec/models/user_spec.rb (hunk 2 above)
Comment (julieahlquist) [reply spec/models/user_spec.rb]: fixed

File: spec/models/user_spec.rb (hunk 1 above)
Comment (julieahlquist) [reply spec/models/user_spec.rb]: fixed

**Review by oliverochman — APPROVED** [review]

**Pull request merged**